pkg/tsdb/testdatasource/scenarios.go

```diff
@@ -191,6 +191,55 @@ Timestamps will line up evenly on timeStepSeconds (For example, 60 seconds means
 	p.queryMux.HandleFunc("", p.handleFallbackScenario)
 }
 
+// addDsFieldConfig decorates field with field config provided from the query editor
+func addDsFieldConfig(field *data.Field, model *simplejson.Json) {
```

Comment:
In Go, initialisms should be written in [consistent case](https://github.com/golang/go/wiki/CodeReviewComments#initialisms):
```suggestion
// addDSFieldConfig decorates field with field config provided from the query editor
func addDSFieldConfig(field *data.Field, model *simplejson.Json) {
```